companies-branches.md

```diff
@@ -0,0 +1,138 @@
+# Companies Branches Modules
+
+## Methods
+
+## Get Companies Branches
+
+The getCompanyBranches method retrieves all branches associated with the current company. It returns an array of [CompanyBranchInterface interfaces](https://github.com/bakaphp/kanvas-core-js/blob/main/src/types/companies-branches.ts#L21).
+
+```js
+Kanvas.companies.getCompanyBranches(): CompanyBranchInterface[]
+```
+
+**Example:**
+
+```js
+const branches = Kanvas.companies.getCompanyBranches();
+```
+
+## Get Company Branches Users
+
+The getCompanyBranchUsers method fetches all users associated with the current company branches. It returns an array of [UserData interfaces](https://github.com/bakaphp/kanvas-core-js/blob/main/src/types/users.ts#L5).
+
+```js
+Kanvas.companies.getCompanyBranchUsers(): UserData[]
+```
+
+**Example:**
+
+```js
+const users = Kanvas.companies.getCompanyBranchUsers();
+```
+
+## Create Company Branch
+
+The createCompanyBranch method is used to create a new company branch. It receives a [CompanyBranchInput interface](https://github.com/bakaphp/kanvas-core-js/blob/main/src/types/companies-branches.ts#L7). It returns a [CompanyBranchInterface interface](https://github.com/bakaphp/kanvas-core-js/blob/main/src/types/companies-branches.ts#L21).
+
+```js
+Kanvas.companies.createCompanyBranch(companyBranchInput: CompanyBranchInput): CompanyBranchInterface
+```
+
+**Example:**
+
+```js
+const newCompanyBranch = Kanvas.companies.createCompanyBranch({
+  name: "My new company branch",
+  companies_id: 1,
+  is_default: false,
+  website: "https://mynewcompanybranch.com",
+  address: "",
+  zipcode: "",
+  email: "",
+  country_code: "do",
+  language: "es",
+  timezone: "",
+  phone: "0000000000",
+  files: [
+    {
+      url: "https://mynewcompanybranch.com",
```

Comment:
@FredPeal make sure images are link to images

Reply:
it's work